src/guide/configuring-labs.md

```diff
@@ -107,11 +109,26 @@ $ python main.py --foo=bar some-value
 
 A very good example to see script parameters in action is MachineLabs [Neural Style Transfer Lab](https://machinelabs.ai/editor/rJQrQ5wjZ/1506415557004-HkTTQ5Dob?file=ml.yaml&tab=editor). 
 
+## Hardware configuration
+
+Obviously, the hardware on which we execute our experiments has a big impact on how much time is being spent on for example training a neural net. That's why we often want to make sure our code is executed on **GPU-accelerated machines**. MachineLabs supports CPU and GPU hardware.
+
+We will open up possible hardware configurations in the future but for now every lab is executed on CPU hardware by default. To run our labs on GPU-accelerated machines, all we have to do is to set the `hardwareType` configuration to `gpu` like this:
```

Comment:
We may want to add here that besides setting the `hardwareType` the user also has to pick a GPU compatible docker image

Reply:
Good catch